frontend/src/lib/components/LemonTable/columnUtils.tsx

```diff
@@ -7,17 +7,18 @@ import { UserBasicType } from '~/types'
 
 export function createdAtColumn<T extends { created_at?: string | null }>(): LemonTableColumn<T, 'created_at'> {
     return {
-        title: 'Created on',
+        title: 'Created',
         dataIndex: 'created_at',
         render: function RenderCreatedAt(created_at) {
             return created_at ? (
-                <div style={{ whiteSpace: 'nowrap' }}>
+                <div style={{ whiteSpace: 'nowrap', textAlign: 'right' }}>
```

Comment:
Whenever you feel yourself reaching to add anything to a style tag, see if you can refactor it with utility classes instead:

```suggestion
                <div className='whitespace-nowrap text-right'>
```

Reply:
got it 🤗